### ISSUE TITLE
tree and permission coloring

### ISSUE THREAD

**Issue by ywpkwon** [avdv/scalals#140]
I'm a `colorls` user but I didn't like it due to not respecting `LS_COLORS`.
I tried to use `scalals` on my Ubuntu and it worked well! Thanks for the project.

Now I'm missing only two things:
- `--tree` option 
![image](https://user-images.githubusercontent.com/18429284/190932667-6f0d9690-4e7a-4ea1-9688-bda72da738d0.png)
- coloring permission/user/byte/etc info 
(colorls)
![image](https://user-images.githubusercontent.com/18429284/190932676-ddedc091-698f-4ebe-a193-ce0312c39b37.png)
(scalals)
![image](https://user-images.githubusercontent.com/18429284/190932690-95766af0-c7fa-4ebc-82f6-9e57f197e961.png)

Are you planning on these features? (or is there already an option for any of these?)

**Comment by avdv** [comment]
Yes, these both sound reasonable to implement. And now that you've raised this to me, I might look into these in the near future...

**Comment by avdv** [comment]
Hi @ywpkwon , 

I have created a draft implementation for `--tree`, see #142.

If you want, you can test it using the binaries here: https://github.com/avdv/scalals/actions/runs/3200391409#artifacts
